server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/rfc/model/CreatedIds.scala

```diff
@@ -0,0 +1,49 @@
+/** **************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ * *
+ * http://www.apache.org/licenses/LICENSE-2.0                 *
+ * *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ * ***************************************************************/
+package org.apache.james.jmap.rfc.model
+
+import org.apache.james.jmap.rfc.model.CreatedIds.{ClientId, ServerId}
+import org.apache.james.jmap.rfc.model.Id.Id
+import play.api.libs.json._
+import be.venneborg.refined.play.RefinedJsonFormats._
+
+case class CreatedIds(value: Map[ClientId, ServerId])
```

Comment:
would using a Seq[Tuple[ClientId, ServerId]] work without the implicit conversion methods?

Reply:
No I tried. Still need an implicit conversion `Seq[(ClientId, ServerId)]` to a JsonObject

Reply:
ok